src/components/WwwFrame/LendMenu/TheLendMenu.vue

```diff
@@ -121,14 +121,11 @@ export default {
 		},
 	},
 	methods: {
-		onOpen() {
-			this.$refs?.mega?.onOpen?.();
-		},
```

Comment:
Interesting that this disappeared...

Reply:
Definitely, but it no longer exists in the mega menu component...